sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Release History
 
-### 4.45.0-beta.1 (Unreleased)
+### 4.45.0 (2023-05-12)
 
 #### Features Added
```

Comment:
UUID fix is missing

Reply:
I suppose it is included under the "Other Changes" heading - **Removed synchronized locking on generating random UUIDs**

Reply:
its in other changes section.